ideascube/library/views.py

```diff
@@ -134,7 +134,8 @@ class SpecimenCreate(CreateView):
     def get_initial(self):
         book = get_object_or_404(Book, pk=self.kwargs['book_pk'])
 
-        return {'book': book, 'location': get_server_name()}
+        return {
+            'book': book, 'location': Configuration('server', 'site-name')}
```

Comment:
The API was overhauled as we discussed, and as part of that I fixed this issue in the latest push to this branch.

Note that `Configuration.get` was just as wrong, it should have been `Configuration.get_value`, which clearly shows that the API wasn't clear at all. :stuck_out_tongue: